src/api/instances.tsx

```diff
@@ -36,14 +36,17 @@ export const fetchInstances = (project: string): Promise<LxdInstance[]> => {
 
 export const createInstance = (
   body: string,
-  project: string,
-  target?: string,
+  targetProject: string,
+  targetCluster?: string,
```

Comment:
Shall we better keep the old names, as they are exactly matching the URL parameter names of the API route that is being called?